routes.php

```diff
@@ -97,6 +97,7 @@
 // Comment interactions
 $router->add('POST', '#^/cocktails/(\d+)-[^/]+/comments$#', [CommentController::class, 'addComment']);
 $router->add('GET', '#^/comments/(\d+)/edit$#', [CommentController::class, 'edit']); // Edit comment
+$router->add('POST', '#^/comments/(\d+)/edit$#', [CommentController::class, 'edit']); // Edit comment

```

Comment:
Whilst that is true, the method used in the CommentController does not rely or call for a GET,  because of how the JS is retrieving data.
This meant that I needed to decide on the uri extension on whether I keep edit or update.
Since we call it as to edit a comment and not to update a comment, the dessicion on my end was to use edit for the POST.

Of course given the changes I need to make to the controller this might change, and we can keep edit for GET and POST for update.